src/search.js

```diff
@@ -1,22 +1,28 @@
-import {ascend, assoc, concat, descend, has, isEmpty, map, prop, sortBy, sortWith} from "ramda"
+import {ascend, assoc, concat, contains, filter, has, isEmpty, map, pipe, prop, sortBy, sortWith} from "ramda"
 import * as diacritics from 'diacritics'
 
 
 export function findParametersAndVariables(parameters, variables, query) {
-  function weightIn(string, substring) {
-    const index = string.indexOf(substring)
-    return index === -1 ? Number.MAX_SAFE_INTEGER : index
-  }
   const normalizedQuery = diacritics.remove(query.toLowerCase())
   const parametersAndVariables = concat(parameters, variables)
   if (isEmpty(normalizedQuery)) {
     return sortBy(prop('name'), parametersAndVariables)
   }
-  return sortWith([
-    ascend(item => weightIn(item.name, normalizedQuery)),
-    descend(item => item.normalizedDescription.includes(normalizedQuery)),
-    ascend(prop('name')),
-  ], parametersAndVariables)
+  const matchesInName = pipe(
+    filter(item => item.name.includes(normalizedQuery)),
```

Comment:
That was [`partition`](http://ramdajs.com/docs/#partition) actually (`splitWhen` stops partitioning at first match). But thanks for the good idea!